### ISSUE TITLE
example 2 is incorrect

### ISSUE THREAD

**Issue by radekatsyserco** [MicrosoftDocs/office-docs-powershell#8278]
it should look like example 4, that command does not take -permanent as a parameter

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 20837451-48fe-1dd4-eaf2-e8296abdf4db
* Version Independent ID: a51a4a6c-11f4-ba4d-46fa-5187e2d99e5f
* Content: [Remove-Mailbox (ExchangePowerShell)](https://docs.microsoft.com/en-us/powershell/module/exchange/remove-mailbox?view=exchange-ps#feedback)
* Content Source: [exchange/exchange-ps/exchange/Remove-Mailbox.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/exchange/exchange-ps/exchange/Remove-Mailbox.md)
* Service: **exchange-powershell**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@radekatsyserco, cmdlet reference topics are single sourced, so some topics (like this one) apply to Exchange 2010 - 2019 and Exchange Online.

The Permanent parameter is clearly marked in the description as being on-prem only. For examples, we typically try to identify when the example only applies to on-prem or cloud (note how Example 4 starts with "In Exchange Online, this example...".

I updated examples 2 and 3 to start with "In on-premises Exchange, this example..."

Thanks for the feedback.

Closing
